homeassistant/components/hassio/__init__.py

```diff
@@ -719,7 +719,7 @@ async def _async_update_data(self) -> dict[str, Any]:
             raise UpdateFailed(f"Error on Supervisor API: {err}") from err
 
         new_data: dict[str, Any] = {}
-        supervisor_info = get_supervisor_info(self.hass)
+        supervisor_info = get_supervisor_info(self.hass) or {}
```

Comment:
When is this None?

Reply:
When hassio has not yet fetched data from SU or SU reported issues